src/utils/map-state.js

```diff
@@ -1,183 +1,105 @@
-import PropTypes from 'prop-types';
 import {PerspectiveMercatorViewport} from 'viewport-mercator-project';
 import assert from 'assert';
 
-// MAPBOX LIMITS
-const MAX_PITCH = 60;
```

Comment:
Going beyond 60 hides the map, this might be unexpected/undesired default behavior.